src/components/DeploymentStatusBadge.vue

```diff
@@ -1,18 +1,42 @@
 <template>
-  <template v-if="deployment.status">
-    <p-tag>
-      <DeploymentStatusIcon :status="deployment.status" />
-      {{ titleCase(deployment.status) }}
-    </p-tag>
-  </template>
+  <p-tag v-if="deployment.status" :class="classes.root" class="deployment-status-badge">
```

Comment:
Removed the template wrapper around the root element here to avoid double-nesting `templates` unecessarily

Reply:
I wish you could just add v-if on the root template tag